cluster/engine.go

```diff
@@ -470,8 +470,14 @@ func (e *Engine) updateClientVersionFromServer(serverVersion string) {
 		e.apiClient.UpdateClientVersion("1.23")
 	case versions.LessThan(serverVersion, "1.13"):
 		e.apiClient.UpdateClientVersion("1.24")
-	default:
+	case versions.LessThan(serverVersion, "1.13.1"):
 		e.apiClient.UpdateClientVersion("1.25")
+	case versions.LessThan(serverVersion, "17.03.1"):
```

Comment:
Just one question, if the `serverVersion` is `17.03.1~ce`, what will return from `versions.LessThan(serverVersion, "17.03.1")`? I think it will return true, am I right? Here is some code from:
```
// compare compares two version strings
// returns -1 if v1 < v2, 1 if v1 > v2, 0 otherwise.
func compare(v1, v2 string) int {
	var (
		currTab  = strings.Split(v1, ".")
		otherTab = strings.Split(v2, ".")
	)

	max := len(currTab)
	if len(otherTab) > max {
		max = len(otherTab)
	}
	for i := 0; i < max; i++ {
		var currInt, otherInt int

		if len(currTab) > i {
			currInt, _ = strconv.Atoi(currTab[i])
		}
		if len(otherTab) > i {
			otherInt, _ = strconv.Atoi(otherTab[i])
		}
		if currInt > otherInt {
			return 1
		}
		if otherInt > currInt {
			return -1
		}
	}
	return 0
}

// LessThan checks if a version is less than another
func LessThan(v, other string) bool {
	return compare(v, other) == -1
}
```